skaha/src/main/java/org/opencadc/skaha/SkahaAction.java

```diff
@@ -98,8 +96,14 @@
 import org.apache.log4j.Logger;
 import org.json.JSONArray;
 import org.json.JSONObject;
+import org.opencadc.auth.StandardIdentityManager;
 import org.opencadc.gms.GroupURI;
+import org.opencadc.gms.IvoaGroupClient;
 import org.opencadc.skaha.image.Image;
+import org.springframework.util.StringUtils;
+
+import static java.util.stream.Collectors.toList;
+import static org.springframework.util.CollectionUtils.isEmpty;
```

Comment:
It would be nice it we didn't have to bring in the springframework jar file to support this command.  But, I don't see a reference to it in build.grade?

Reply:
It is shown in the import suggestion, That's why I have used.
Currently I have removed it and added a method to check if the collection is empty or not